GitLab CVE Scan CI (#2)
deckhouse/modules-gitlab-ci: docs/CVE_Scan.gitlab-ci.md

# CVE_Scan CI Integration

## Description

> This ci file will run Trivy CVE scan against module image and its submodules images and then upload reports to defectDojo.

Reviewer: ```suggestion
This CI file will run a Trivy CVE scan against the module images and its submodule images, and then upload the reports to DefectDojo.
```

## Variables

### Job level
```

> IMAGE - full path for module image. e.g.: regestryType.deckhouse.io/path/to/module

Reviewer: ```suggestion
IMAGE - full path to a module image. e.g.: regestryType.deckhouse.io/path/to/module
```

TAG - module image tag
MODULE_NAME - module name

> DECKHOUSE_PROD_REGISTRY - must be deckhouse read registry, used to get trivy databases

Reviewer: ```suggestion
DECKHOUSE_PROD_REGISTRY - must be deckhouse read registry, used to get trivy databases
```

probably it's better to use something like trivy registry or trivy db or something like this

Reviewer: but we can go with deckhouse also
